### ISSUE TITLE
Connecting to Oracle Internet Directory server

### ISSUE THREAD

**Issue by schmitzkr** [dbeaver/dbeaver#7702]
#### System information: 
- Operating system (distribution) and version: Linux and Windows
- DBeaver version 6.3.3

#### Connection specification:
- Database name and version Oracle 11g

#### Describe your question:
I am currently using TOAD for Oracle to connect to several (100+) oracle databases.  Within TOAD we can configure OID connections using LDAP.ora configuration files which hold all the necessary details to connect to those OID servers. And then the OID server has a nice listing of all the databases available.  I am trying to make the switch to dbeaver and I cannot seem to configure these OID servers to give me that nice up to date listing of all the databases.
I can connect to all of these databases with the TNS connection type. 
Am I missing something in the configuration/settings that allows me to use an LDAP connection type?  Or do I need to continue to rely on TNS names?
Thanks!

**Comment by CoyoteProd** [comment]
Same need here...
I want to switch from SQL Developper...
But DBeaver seems not able to Handle "ldap.ora" file
(We plan to replace about 200 installations of SQL Developper in our institution...)

**Comment by mquintas** [comment]
maybe you can try to connect using the custom jdbl url, for example

`jdbc:oracle:thin:@ldap://your_ldap_server:389/your_service_name,cn=OracleContext,dc=WORLD`

I'm not an oracle expert, but that worked for me, maybe it dependes on how the services are configured in the ldap/oracle server of your organization.
in my case "your_ldap_server" is what I have in my ldap.ora and "your_service_name" is the service name that the admins provided me.

Also, no sure if its mandatory, but maybe you need the oracle client installed and configured in the connection settings (above the security section).

**Comment by schmitzkr** [comment]
I don't think we can simply use a custom JDBC URL.  The OID server is serving up several database servers and service names within each one.
I can use a JDBC URL to connect to a single database and service.  But in this case, the LDAP support in TOAD allows me to connect to a OID server, and it will list all service names found there, then i can chose the one i want.  I would need to create hundreds of custom JDBC urls, one for each service i may need to access.
Ideally the oid server would be available before i create a database connection, and while creating a new connection i could select from the servers/services listed there to create the new connection.
